core/storage/config/gae_models.py

```diff
@@ -189,10 +189,14 @@ class FeatureFlagConfigModel(base_models.BaseModel):
     The id is the name of the feature-flag.
     """
 
+    # If the feature flag is force enable for all the users.
     force_enable_for_all_users = datastore_services.BooleanProperty(
         default=False, indexed=True)
+    # The percentage of logged-in users for which the feature flag will
+    # be enabled. Ranges between 0 to 100.
```

Comment:
Ranges between 0 to 100 --> The value of this field should be between 0 and 100.

Reply:
Done